agent-deploy/Dockerfile

```
@@ -104,6 +105,30 @@ COPY .curlrc .wgetrc /root/
 RUN git config --global user.email "package@datadoghq.com" && \
     git config --global user.name "Bits"
 
+# Install Mono
+RUN gpg --no-default-keyring --keyring /usr/share/keyrings/mono-official-archive-keyring.gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF \
+    && echo "deb [signed-by=/usr/share/keyrings/mono-official-archive-keyring.gpg] https://download.mono-project.com/repo/ubuntu stable-bionic main" | tee /etc/apt/sources.list.d/mono-official-stable.list \
+    && chmod 644 /usr/share/keyrings/mono-official-archive-keyring.gpg \
+    && apt update \
+    && apt install -y mono-devel
+
+# Install .NET
+RUN wget https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb \
```

```suggestion
RUN UBUNTU_VERSION=$(cat /etc/os-release | grep VERSION_ID | cut -d = -f 2 | xargs) && \
wget https://packages.microsoft.com/config/ubuntu/${UBUNTU_VERSION}/packages-microsoft-prod.deb -O packages-microsoft-prod.deb \
```
This way we can always get the correct package when we decide to bump the base image